### ISSUE TITLE
Cart and Checkout Blocks: Fix cursors for radio and checkbox inputs

### ISSUE THREAD

**Issue by ralucaStan** [woocommerce/woocommerce-gutenberg-products-block#8424]
## Describe the bug

All the checkbox and radio inputs used in the Cart and Checkout blocks don't have the correct cursor for hover states.

## To reproduce

1. Go to the Checkout Block, and make sure you have shipping and payments declared
2. Hover the checkbox input for billing and shipping addresses. Hover the label next to it.
3. Notice that the cursor does not change.
4. Scroll down to the shipping section and hover over one shipping radio input. Hover the corresponding label.
5. Notice the cursor does not change

## Expected behavior

The cursor should change to a pointer when hovering over a checkbox & radio input, as well as over their corresponding labels.
Disabled inputs and their labels should have the default cursor.

**Comment by Sidsector9** [comment]
@nielslange I would like to work on this.

**Comment by ankitguptaindia** [comment]
I would like to do QA on this issue/PR.

**Comment by ralucaStan** [comment]
@adrianlbs heads up that this issue has been closed

**Comment by adrianlbs** [comment]
Awesome. Thanks for the update, @ralucaStan !
